local-cli/runIOS/runIOS.js

```diff
@@ -27,15 +27,24 @@ function runIOS(argv, config) {
 }
 
 function _runIOS(argv, config, resolve, reject) {
-  const args = parseCommandLine([{
-    command: 'simulator',
-    description: 'Explicitly set simulator to use',
-    type: 'string',
-    required: false,
-    default: 'iPhone 6',
-  }], argv);
+  const args = parseCommandLine([
+    {
+      command: 'simulator',
+      description: 'Explicitly set simulator to use',
+      type: 'string',
+      required: false,
+      default: 'iPhone 6',
+    },
+    {
+      command: 'ios-project-path',
```

Comment:
@mkonicek I'd disagree with `ios-project-path`, because this is not something that wouldn't make sense on Android (or other platform) one day. Also the command is `run-ios` which already sets the context for `ios`. `simulator` param to this command is also iOS-specific but it doesn't have the prefix.

Let's stick to `project-path`?